1-exercises/F-array-forEach/exercise.js

```diff
@@ -1,13 +1,27 @@
-/* 
+/*
   Using .forEach() print the numbers 1 to 15, with some exceptions:
-  - For multiples of 3 print “Fizz” instead of the number 
-  - For the multiples of 5 print “Buzz”. 
+  - For multiples of 3 print “Fizz” instead of the number
+  - For the multiples of 5 print “Buzz”.
   - For numbers which are multiples of both 3 and 5 print “FizzBuzz”
 
   An array with numbers 1-15 has been provided.
 */
 
 let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+arr.forEach((element) => {
+  if (element % 3 === 0 && element % 5 !== 0) {
+    console.log("Fizz");
+  }
+  if (element % 5 === 0 && element % 3 !== 0) {
+    console.log("Buzz");
+  }
+  if (element % 3 === 0 && element % 5 === 0) {
+    console.log("FizzBuzz");
+  }
+  if (element % 3 !== 0 && element % 5 !== 0) {
+    console.log(element);
+  }
+});
```

Comment:
Yes, this works well. Another option would be to build up a string, so that you can concatenate "Fizz" and "Buzz". Or check the "FizzBuzz" case first.